### ISSUE TITLE
Turbolinks breaks links inside SVG

### ISSUE THREAD

**Issue by mezis** [turbolinks/turbolinks#110]
Any link within an (inline) SVG element breaks, and following the link attempts to redirect to the url `[object SVGAnimatedString]` (which obviously doesn't work).

This for instance, doesn't work if Turbolinks 5 is loaded (checked with beta2 and beta5).
```html
<svg width="100" height="100">
  <text x="0" y="50">
    <a href="https://google.com/">broken link</a>
  </text>
</svg>
```
Here's a corresponding [JSfiddle](https://jsfiddle.net/mezis/683hgt0y/2/).
My Javascript-fu is unfortunately not good enough to propose a fix, but hopefully this still helps!

**Comment by mezis** [comment]
This kludgy workaround fixed the issue for me, in case someone bumps the same issue:

```coffee
$('svg a').on 'click', (e) ->
  Turbolinks.visit $(e.target).attr('href')
  false
```

**Comment by packagethief** [comment]
Thanks for the detailed report, @mezis! Fixed in fe41ac3.
